src/app/app.component.ts

```diff
@@ -10,25 +16,95 @@ import { AuthService } from './auth/auth.service';
 export class AppComponent {
   title = 'altrusight';
   sidebarOpened: boolean;
+  allNonProfitsNames: string[] = [];
+
+  // searchInput = new FormControl();
+  // filteredNonProfits: Observable<string[]>;
+
 
   // TODO: issue with auth --> auth service is not instantiated until AFTER app component is
   // so we will always have auth issues on first render until fixing this
   // https://stackoverflow.com/questions/35191617/how-to-run-a-service-when-the-app-starts-in-angular-2
-  constructor(public router: Router, public authService: AuthService) {
+  constructor(public router: Router, public authService: AuthService, private nonProfitsService: NonprofitsService) {
       this.sidebarOpened = this.authService.isLoggedIn;
+      this.nonProfitsService.getNonprofit('')
```

Comment:
This line does nothing since we're not subscribing or setting a variable, also I think similar behavior can be had from getNonprofits(). That method does need to be changed though.

Reply:
Never mind, I see correct use below this. Just remove this line, I guess